components/gallery/List.vue

```diff
@@ -0,0 +1,303 @@
+<script>
+
+import _dropRight from "lodash/dropRight"
+import _takeRight from "lodash/takeRight"
+
+export default {
+    name: "List",
+    props: {
+        items: {
+            type: Array,
+            default: () => []
+        },
+    },
+    data() {
+        return {
+            isImageLoaded: false,
+            isVideoLoaded: false,
+            isImageError: false,
+            isVideoError: false,
+            caption: '',
+            image: null
+        }
+    },
+    computed: {
+        imageClasses() {
+            return [
+                {
+                    'has-loaded-media': this.isImageLoaded || this.isVideoLoaded,
+                    'has-full-media-error': this.isVideoError && this.isImageError && !this.isImageLoaded && !this.isVideoLoaded,
+                }
+            ]
+
+        },
+        preparedList() {
+            let tempData = this.items
+            let {right, left} = this.shapeListData(tempData, 3)
+            let trimmed = this.shapeListData(left, 2)
+
+            return [
+                [
+                    {
+                        listItems: trimmed.left
+                    },
+                ],
+                [
+                    {
+                        listItems: trimmed.right
+                    },
+                    {
+                        listItems: right
+                    }
+                ]
+
+            ]
+        },
+    },
+    methods: {
+        videoLoaded() {
+            this.isVideoLoaded = true
+        },
+        imageLoaded() {
+            this.isImageLoaded = true
+        },
+        videoError() {
+            this.isVideoError = true
+        },
+        imageError() {
+            this.isImageError = true
+        },
+
+        mouseOver(item) {
+            this.caption = item.featuredImage.node.caption !== null ? item.featuredImage.node.caption : `Who the fuxt is ${item.title}?`
+            this.image = item.featuredImage.node
+        },
+        mouseOut() {
+            this.isVideoLoaded = false
+            this.isImageLoaded = false
+            this.isVideoError = false
+            this.isImageError = false
+            this.caption = ''
+            this.image = null
+        },
+        shapeListData(array, offset) {
+            let count = array.length
+            let trimLastCount = Math.floor(count / offset)
+            let rightArray = _takeRight(array, trimLastCount)
+            let leftArray = _dropRight(array, trimLastCount)
+
+            return {
+                left: leftArray,
+                right: rightArray
+            }
+        },
+    }
+
+}
+</script>
+
+
+<style lang="scss" scoped>
+// import font at the component level because it doesn't appear to be loaded globally
+@font-face {
+    font-family: 'RM Neue';
+    src: url('../../assets/fonts/RMNeue-Regular.woff2') format('woff2'),
+    url('../../assets/fonts/RMNeue-Regular.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
```

Comment:
Read this for the correct way we use fonts https://github.com/funkhaus/fuxt/wiki/Fonts

Reply:
Ok this got pretty gross. `storybook` was breaking because it couldn't resolve the alias `~` to the font files `src: url("~/assets/fonts/RMNeue-Regular.woff2") format("woff2"),`. After google search hell I pulled down `fuxt` from your repo and tried setting up the custom font in that project and everything worked as expected so ended up replacing the programming-partners `package.json` and the `nuxt.config.js` with the ones from the `fuxt` repo and it works. I know this isn't ideal but I was going crazy and just wanted to move on.